### ISSUE TITLE
gosimple: go get fails due to "undefined: sort.Slice"

### ISSUE THREAD

**Issue by glerchundi** [dominikh/go-tools#61]
Yesterday worked perfectly, it seems like there was a change in between:

Using golang 1.7.5 in our CI environment:
```
go get -u honnef.co/go/tools/cmd/gosimple
# honnef.co/go/tools/lint
/go/src/honnef.co/go/tools/lint/lint.go:217: undefined: sort.Slice
```

regards,

**Comment by dominikh** [comment]
Closed by 75e85735cca94e6aa5a598e9c549ca37096e7d9e.

**Comment by glerchundi** [comment]
That was really quick, thanks @dominikh. Can confirm it works, green again.
